.github/workflows/ci.yml

```diff
@@ -42,12 +42,12 @@ jobs:
         java-package: jre
     - run: npm install
     - name: Use Dumper
-      run: |
-        mkdir output
-        node bin/dumpPackets -v ${{ matrix.mcVersionIndex }} -o output/${{ matrix.mcVersionIndex }}
-        
+      run: node bin/dumpPackets -v ${{ matrix.mcVersionIndex }}
+    - name: Save PR number
+      run: echo ${{ github.event.number }} > ./output/NR
```

Comment:
this is going to write several times to the same file, putting it in build could do the trick (you can have another upload artifact pushing to the same artifact name there)